examples/full-jaffle-shop-demo/dbt/models/plan.yml

```diff
@@ -3,4 +3,22 @@ models:
   - name: plan
     columns:
       - name: id
-      - name: plan_name
\ No newline at end of file
+      - name: plan_name
+      - name: metadata
+        meta:
+          dimension:
+            hidden: true
```

Comment:
what if this dimension has custom sql? will that still work?

also.. I can't think of a different type than json and arrays for base dimension. is there another usecase for this?

Reply:
> what if this dimension has custom sql? will that still work?

it still works

> is there another usecase for this?

There are a few use cases. You can see more in the docs https://docs.lightdash.com/references/dimensions#additional-dimensions